### PR TITLE
Cas 415/media attachments with text vh

### PR DESCRIPTION
**Pull request by ogkuzmin**
### Description

ViewHolder for plain text with media attachments

text+media_vh1

### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [ ] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] Comparison screenshots added for visual changes
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewTypeMapper.kt [hunk 1]
```diff
@@ -5,19 +5,20 @@ import com.getstream.sdk.chat.model.ModelType
 import io.getstream.chat.android.client.models.Attachment
 
 internal object MessageListItemViewTypeMapper {
-    fun getViewTypeValue(messageListItem: MessageListItem): Int = listItemToViewType(messageListItem).typeValue
+    fun getViewTypeValue(messageListItem: MessageListItem): Int = listItemToViewType(messageListItem).ordinal
```
Comment (zsmb13) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewTypeMapper.kt:8]: Nicely done, glad we made this change 👍

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewType.kt [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+    DATE_DIVIDER,
+    MESSAGE_DELETED,
+    PLAIN_TEXT,
+    REPLY_MESSAGE,
+    PLAIN_TEXT_WITH_FILE_ATTACHMENTS,
+    PLAIN_TEXT_WITH_MEDIA_ATTACHMENTS,
+    MEDIA_ATTACHMENTS,
+    ATTACHMENTS,
+    LOADING_INDICATOR,
+    THREAD_SEPARATOR
```
Comment (zsmb13) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewType.kt:13]: Nit: can we please add a trailing comma here? 😄

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewTypeMapper.kt [hunk 3]
```diff
@@ -5,19 +5,20 @@ import com.getstream.sdk.chat.model.ModelType
...
+            MessageListItemViewType.DATE_DIVIDER.ordinal -> MessageListItemViewType.DATE_DIVIDER
+            MessageListItemViewType.MESSAGE_DELETED.ordinal -> MessageListItemViewType.MESSAGE_DELETED
+            MessageListItemViewType.PLAIN_TEXT.ordinal -> MessageListItemViewType.PLAIN_TEXT
+            MessageListItemViewType.REPLY_MESSAGE.ordinal -> MessageListItemViewType.REPLY_MESSAGE
+            MessageListItemViewType.PLAIN_TEXT_WITH_FILE_ATTACHMENTS.ordinal -> MessageListItemViewType.PLAIN_TEXT_WITH_FILE_ATTACHMENTS
+            MessageListItemViewType.PLAIN_TEXT_WITH_MEDIA_ATTACHMENTS.ordinal -> MessageListItemViewType.PLAIN_TEXT_WITH_MEDIA_ATTACHMENTS
+            MessageListItemViewType.MEDIA_ATTACHMENTS.ordinal -> MessageListItemViewType.MEDIA_ATTACHMENTS
+            MessageListItemViewType.ATTACHMENTS.ordinal -> MessageListItemViewType.ATTACHMENTS
+            MessageListItemViewType.LOADING_INDICATOR.ordinal -> MessageListItemViewType.LOADING_INDICATOR
+            MessageListItemViewType.THREAD_SEPARATOR.ordinal -> MessageListItemViewType.THREAD_SEPARATOR
```
Comment (zsmb13) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewTypeMapper.kt:21]: I think this is the same as

```kotlin
return MessageListItemViewType.values().find { it.ordinal == viewTypeValue }
            ?: error("View type must be a value from MessageListItemViewType")
```

**Review by zsmb13 — DISMISSED** [review]
Looks great! Still loving these shape APIs that we're using here.

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewTypeMapper.kt (hunk 3 above)
Comment (ogkuzmin) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/MessageListItemViewTypeMapper.kt:21]: Yeah, it's more concise

**Review by zsmb13 — APPROVED** [review]

**Pull request merged**